book/website/collaboration/inclusive-events/inclusive-events-schedules.md

```diff
@@ -0,0 +1,206 @@
+
+> **Example 1: 1 hour Focus Group Event**
+    - 10 minute welcome, Code of Conduct, and Session introduction
+    - 40 minute Discussion between participants
+    - 10 minute break or 10 minute thank you and session close
+
+> **Example 2: 1 hour Workshop event**
+    - 5 minute introduction 
+    - 20 minute session
+    - 5 minute break
+    - 20 minute session
+    - 10 minute break or 10 minute thank you and session close
+
+It is best to get feedback and input from a representative with lived experience when planning your session timings. This is because there may be diferences in what is required for a specific community that would negatively impact a different set of individuals. 
+
+For example, if your event invites specifically people with lived experience of ADHD (and no other cognitive impairments) for a creative workshop. You may want to consider offering longer sessions that are facilitated with self-directed breaks. For example, for a 1.5hr event you may consider a 10 minute introducion, followed by a 50 minute self directed session, a 15 minute break, and then a 15 minute conclusion or share out and session close. This is because many people with a lived experience of ADHD can find that forced interruption is very disruptive and prohibitive of meaningful engagement on tasks that are creative or require deep focus.
+
+### Event Duration
+In addition to session length, it is also important to consider how your event will be structured to support the involvement and engagement of people with lived experience. 
+
+If you are organizing a multi-day conference, consider picking either a single day or a repeating timeslot that features any content specifically important for people with lived experience. For example, Day 1 of a 3-Day event might schedule the majority of sessions focused on public engagement and lived experience, followed by Day 2 which could focus on technical workshops and other content. Alternatively, sessions from 10 - 1pm on all 3 days could focus on lived experience content. 
+
+This can support the engagement of people with lived experience who volunteer their time to engage in research, as well as people with lived experience of disabilities and medication conditions that can significantly impact their energy levels. 
+
+
+## Event Content: Planning for Engagement
+When identifying the content of your event, you should spend time identifying what type of content and sessions will attendees with lived experience be interested in, what "engagement" means for your specific event, and how will you ensure that attendees with lived experience are able to engage in your event.
```

Comment:
When identifying the content of your event, you should spend time identifying what type of content and sessions ~~will~~ attendees with lived experience **will** be interested in, what "engagement" means for your specific event, and how ~~will~~ you **will** ensure that attendees with lived experience are able to engage in your event.